lms/static/js/student_account/multiple_enterprise.js

```diff
@@ -0,0 +1,50 @@
+(function(define) {
+    'use strict';
+    define(['jquery', 'jquery.cookie'], function($) {
+        var MultipleEnterpriseInterface = {
+
+            urls: {
+                learners: '/enterprise/api/v1/enterprise-learner/'
+            },
+
+            headers: {
+                'X-CSRFToken': $.cookie('csrftoken')
+            },
+
+            /**
+             * Fetch the learner data, then redirect the user to a enterprise selection page if multiple
+             * enterprises were found.
+             * @param  {string} user.
+             * @param  {string} redirectUrl The URL to redirect to.
+             */
+            check: function(user, redirectUrl, nextUrl) {
+                var next = nextUrl || '/';
+                $.ajax({
```

Comment:
i'm not thrilled to be adding more calls to the enterprise learner endpoint - it already gets throttled sometimes. is it possible to get this info into the cookie (jwt cookie would be ideal but i understand that may be too much at this point)? or can we increase the rate limit on the enterprise learner endpoint and monitor closely after we release this?

Reply:
Agreed. We will increase the rate limit on the enterprise learner endpoint and monitor closely after release.

Reply:
[Ticket](https://openedx.atlassian.net/browse/ENT-2466) created.

Reply:
As per my understanding this increase in load on the `enterprise-learner` endpoint can't be controlled via individual user throttling limits, since these requests will be initiated by a single user who has just logged in and it is highly unlikely that they will be logging in at a rate greater than `60/min`.